### PR TITLE
Using displayName for icon accessibility

### PR DESCRIPTION
**Pull request by ianbjacobs**
Addresses #127 
Guidance to use the displayName to ensure accessibility.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/secure-payment-confirmation/pull/162.html" title="Last updated on Nov 12, 2021, 10:32 PM UTC (b6423ce)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/secure-payment-confirmation/162/9ec6e5a...b6423ce.html" title="Last updated on Nov 12, 2021, 10:32 PM UTC (b6423ce)">Diff</a>

### CODE REVIEW

File: spec.bs [hunk 1]
```diff
@@ -1257,6 +1259,8 @@ with a random salt, making them non-consistent across calls.
 
 # Accessibility Considerations # {#sctn-accessibility-considerations}
 
+User agents render the {{PaymentCredentialInstrument/icon}} and {{PaymentCredentialInstrument/displayName}} together. Relying parties ensure the accessibility of the icon presentation by providing sufficient information via the {{PaymentCredentialInstrument/displayName}}. For example, if the icon represents a bank, include the bank name in the {{PaymentCredentialInstrument/displayName}}.
```
Comment (stephenmcgruer) [inline spec.bs:1262]: Do you intend this sentence to be descriptive for RPs ("Relying parties ensure the accessibility of [...] by [...]"), or should it be imperative ("Relying Parties should ensure ...")?

Comment (ianbjacobs) [reply spec.bs:1262]: My intention is to be descriptive. (The conformance section aims at software implementations.)

Comment (stephenmcgruer) [reply spec.bs:1262]: Ack, in which case you might want to change 'include the bank name' to 'by including the bank name' ?

Comment (stephenmcgruer) [reply spec.bs:1262]:
```suggestion
User agents render the {{PaymentCredentialInstrument/icon}} and {{PaymentCredentialInstrument/displayName}} together. Relying parties ensure the accessibility of the icon presentation by providing sufficient information via the {{PaymentCredentialInstrument/displayName}}. For example, if the icon represents a bank, by including the bank name in the {{PaymentCredentialInstrument/displayName}}.
```

**Review by stephenmcgruer — APPROVED** [review]

**Comment by RealJoshue108** [comment]
This seems sensible from an #a11y perspective, but I would like to see an implementation to be sure. @ianbjacobs

**Pull request merged**